### ISSUE TITLE
Use separate attrs for select and text form field to help with accessibility

### ISSUE THREAD

**Issue by KunjPrasad** [stefanfoulis/django-phonenumber-field#460]
I am using `PhoneNumberPrefixWidget` ([link](https://github.com/stefanfoulis/django-phonenumber-field/blob/main/phonenumber_field/widgets.py#L49)). I want to provide different widget attributes for phone country code select and phone number text fields. A primary goal is to provide different `aria-label` attributes for the two input type because I'm unable to identify how to put different label for each and position it side by side since it is a `MultiWidget`. Not sure if there are alternate ways to make it accessible.

Looking at other example of `MultiWidget` in Django (like, [here](https://github.com/django/django/blob/main/django/forms/widgets.py#L897)), it seems that different `attrs` kwargs are defined.

**Comment by francoisfreitag** [comment]
Should have been resolved with #502.
